libraries/apollo-normalized-cache/src/commonMain/kotlin/com/apollographql/apollo3/cache/normalized/ApolloStore.kt

```diff
@@ -75,14 +75,30 @@ interface ApolloStore {
    * @param publish       whether to publish the changed keys to listeners
    * @return the changed keys
    */
-  fun <D : Operation.Data> writeOperation(
+  suspend fun <D : Operation.Data> writeOperation(
       operation: Operation<D>,
       operationData: D,
       customScalarAdapters: CustomScalarAdapters = CustomScalarAdapters.Empty,
       cacheHeaders: CacheHeaders = CacheHeaders.NONE,
       publish: Boolean = true,
   ): Set<String>
 
+  /**
+   * Write an operation data to the store and optionally publish changes of [Record] which have changed,
+   * that will notify any watcher that depends on these [Record] to re-fetch.
```

Comment:
```suggestion
   * Write an operation data to the store.
```

Or maybe do we want to highlight it a bit more? 

```suggestion
   * Write an operation data to the store. Call [publish] with the returned keys to notify any watcher that depends on these [Record] to re-fetch.
```

(same for `writeFragment`)

Reply:
Good call out. I added a `@see publish` tag so hopefully it should be enough.